support/developer/biztalk/management-operations/maintain-troubleshoot-database.md

```diff
@@ -31,56 +33,56 @@ Microsoft BizTalk Server databases and the health of the databases are important
 
 This article describes how to maintain BizTalk Server databases and how to troubleshoot BizTalk Server database issues.
 
-## You must disable the auto update statistics and auto create statistics options
+## You must disable the `auto update statistics` and `auto create statistics` options
 
-You must keep the `auto create statistics` and `auto update statistics` options disabled on the `BizTalkMsgBoxDb` database. To determine whether these settings are disabled, execute the following stored procedures in SQL Server:
+You must keep the `auto update statistics` and `auto create statistics` options disabled on the `BizTalkMsgBoxDb` database. To determine whether these settings are disabled, execute the following stored procedures in SQL Server:
 
 ```sql
-exec sp_dboption 'BizTalkMsgBoxDB', 'auto create statistics'
-exec sp_dboption 'BizTalkMsgBoxDB', 'auto update statistics'
+EXEC sp_dboption 'BizTalkMsgBoxDB', 'auto create statistics'
+EXEC sp_dboption 'BizTalkMsgBoxDB', 'auto update statistics'
 ```
 
-You should set the CurrentSetting setting to off. If this setting is set to on, turn it off by executing the following stored procedures in SQL Server:
+You should set the current setting to `OFF`. If this setting is set to `ON`, turn it off by executing the following stored procedures in SQL Server:
 
 ```sql
-exec sp_dboption 'BizTalkMsgBoxDB', 'auto create statistics', 'off'
-exec sp_dboption 'BizTalkMsgBoxDB', 'auto update statistics', 'off'
+EXEC sp_dboption 'BizTalkMsgBoxDB', 'auto create statistics', 'off'
+EXEC sp_dboption 'BizTalkMsgBoxDB', 'auto update statistics', 'off'
 ```
 
 ## You must set the Max Degree of Parallelism property correctly
 
-On the computer that is running SQL Server and hosting the `BizTalkMsgBoxDb` database, set the Max Degree of Parallelism `run_value` and `config_value` properties to a value of **1**. To determine the Max Degree of Parallelism setting, execute the following stored procedure against the Master database in SQL Server:
+On the computer that is running SQL Server and hosting the `BizTalkMsgBoxDb` database, set the max degree of parallelism `run_value` and `config_value` properties to a value of **1**. In later SQL versions, it's also possible to specify this setting per database instead of per SQL instance. To determine the Max Degree of Parallelism setting, execute the following stored procedure against the Master database in SQL Server:
```

Comment:
**Max Degree of Parallelism**

See https://learn.microsoft.com/en-us/sql/database-engine/configure-windows/configure-the-max-degree-of-parallelism-server-configuration-option?view=sql-server-ver16#SSMSProcedure